src/index.ts

```diff
@@ -0,0 +1,46 @@
+import * as _ from 'lodash';
```

Comment:
This doesn't seem to be used. It's also not listed in `package.json`. Unintentional auto-import?